src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -3314,12 +3314,16 @@ class ArnoldGlobals
 					}
 					break;
 				}
-				case IECoreScenePreview::Renderer::SceneDescription :
+				case IECoreScenePreview::Renderer::SceneDescription : {
 					// An ASS file can only contain options to render from one camera,
```

Comment:
Has SolidAngle now decided they want to move away from the name "ass"?  If so, should we change this comment as well?

Reply:
The `AiSceneWrite()` function is more general than `AiASSWrite()` in that it supports writing to other formats, one of them being USD. I don't know if this means they intend to drop ASS support at some point and use USD instead though.

A handy side effect of this change is that the ArnoldRender node can actually write USD files now if you enter a filename ending in `.usda`.

Reply:
In any case, I've updated the comment in https://github.com/GafferHQ/gaffer/pull/4300/commits/39a77440b068a11aa753e9b28c5b033e558752ba.